.github/workflows/environments.yaml

```
@@ -0,0 +1,38 @@
+name: Tests of conda environments on both Linux and Mac
+
+on:
+    push:
+        branches:
+            - main
+            - develop
+    pull_request:
+
```

Would be worth limiting this to only run when there have been changes in the relevant files.
Even better would be to skip matrix items that don't have file changes, but that's more complex (and relies on another third-party plugin).

```suggestion
on:
    push:
        branches:
            - main
            - develop
        paths:
           - envs/*.yaml
           - templates/environment.yaml
           - requirements-*.yaml
    pull_request:
        paths:
           - envs/*.yaml
           - templates/environment.yaml
           - requirements-*.yaml
```

---

Looks nice, great suggestion! The doc says the checks will stay in 'pending' in case path filters kick in, which is maybe not so nice. Anyways, let's give this a try and we can always remove it if we don't like it.